app/views/exam_templates/_form.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= form_for @exam_templates,
+             as: :exam_template,
+             url: assignment_exam_templates_path,
+             html: {multipart: true} do |t| %>
+    <div class='template-pane'>
+      <h4><%= link_to "Download Exam Template", download_assignment_exam_templates_path %></h4><br>
```

Comment:
Let's make this a bit better. Pass in `id: exam_template.id` to the path helper here, and make the `download` route a `member` instead of `collection` (in the routes.rb file), and then update the `download` method in the controller to look up a specific exam template based on the `params[:id]`.